src/Telemetry/Library/Telemetry.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
My understanding is that this is going to be distributed as a nuget package?
If so I believe it's missing a lot of properties (like Author, Copyright, PackageIcon, etc).
The project is also missing the AssemblyConstants needed for DelaySigning.

Reply:
As of now, this will be distributed together with the Q# Formatter via project reference.

But I'll add the required metadata.

Reply:
The common assembly metadata and delay signing info is automatically added via [/src/Common/AssemblyCommon.props](https://github.com/microsoft/qsharp-compiler/blob/main/src/Common/AssemblyCommon.props) and [/src/Common/DelaySign.props](https://github.com/microsoft/qsharp-compiler/blob/main/src/Common/DelaySign.props) at the root of the repo.